model/Core/Vocabularies/RelationshipType.md

```diff
@@ -66,7 +66,7 @@ Build Profile specific RelationshipType descriptions can be found [here](https:/
 - buildInvokedBy: Agent that invoked the build
 - buildOnBehalfOf: Actor for which buildInvokedBy is acting on behalf of
 - buildHostOf: Element in which the build instance runs on
-- affected: (Security/VEX) Designates an elemented as affected by a vulnerability
-- notAffected: (Security/VEX) Specifies a vulnerability has no impact on an element
-- fixed: (Security/VEX) A vulnerability has been fixed in an element
-- underInvestigation: (Security/VEX) The impact of a vulnerability is being investigated
+- affects: (Security/VEX) Designates an elemented as affected by a vulnerability
```

Comment:
This direction can also be used, to describe multiple affected versions or artifacts at the same time.

Reply:
Thanks Max good catch :pray: 
I've updated the descriptions to call for one or more elements.